test/modules/aggregation-layers/contour-layer/marching-squares.spec.js

```diff
@@ -393,6 +450,15 @@ const GETVERTEX_TESTS = [
   }
 ];
 
+const GETLINEARINTERPOLATION_TESTS = [
+  {
+    min: 10,
+    max: 80,
+    threshold: 50,
+    li: 0.571
```

Comment:
is this in [-.5, .5] ?

Reply:
No its 0 to 1

Reply:
I have another function named getLinearOffsetValue which convert the range [0 , 1]
to [-0.5, 0.5]
```
function getLinearOffsetValue(offset, weights, threshold) {
  const linearInterpolation = getLinearInterpolation(
    weights[offset[0]],
    weights[offset[1]],
    threshold
  );
  // linearInterpolation is between 0 and 1, to match with (-0.5, 0.5) subtracting with 0.5
  return linearInterpolation - 0.5;
}
```